src/sentry/tasks/summaries/daily_summary.py

```diff
@@ -209,3 +229,40 @@ def build_summary_data(
         fetch_key_performance_issue_groups(ctx)
 
     return ctx
+
+
+def build_top_projects_map(context: OrganizationReportContext):
+    """
+    Order the projects by which projects have the highest error count for the day
+    """
+    projects_by_error_total = {
+        project_id: context.total_today
+        for project_id, context in context.projects_context_map.items()
+    }
+    top_projects = [
+        k
+        for k, v in sorted(projects_by_error_total.items(), key=lambda item: item[1], reverse=True)
+    ]
+    top_projects_context_map = {}
+    # for now, hard code to the top 2 projects
+    for project in top_projects[:2]:
+        project_context = context.projects_context_map[project]
+        top_projects_context_map[project] = project_context
+
+    return top_projects_context_map
+
+
+def deliver_summary(ctx: OrganizationReportContext, users: list[int]):
+    # TODO: change this to some setting for daily summary
+    user_ids = notifications_service.get_users_for_weekly_reports(
+        organization_id=ctx.organization.id, user_ids=users
+    )
+    for user_id in user_ids:
+        top_projects_context_map = build_top_projects_map(ctx)
```

Comment:
@ceorourke can't we do this outside of the for lop? The context is the same for each user.